drivers/hmis_external_apis/app/assets/javascripts/external_forms.js

```diff
@@ -111,29 +111,59 @@ $(function () {
   });
 });
 
-window.addDependentGroup = function (inputName, condValue, targetSelector) {
-  var target = $(targetSelector);
-  var show = function () {
+// conditions: [{input_name: 'name', input_value: 'value'}, ...]
+// targetSelector: selector for the item that is conditionally shown
+// enableBehavior: 'ANY' or 'ALL' conditions must be met to show the target selector
+window.addDependentGroup = function (conditions, targetSelector, enableBehavior = 'ANY') {
+  const target = $(targetSelector); // the item with enable_when on it
```

Comment:
Change back to `var` for highest compatibility with ios strict mode, and others

Reply:
Totally, my comment was a note-to-self to switch back to `var`. So I think we are in agreement